scripts/dockerfile-config/static-nginx.conf

```diff
@@ -6,6 +6,8 @@ server {
     location / {
         root   /usr/share/nginx/html;
         index  index.html index.htm;
+        # The following is necessary for React router based apps
+        try_files $uri $uri/ /index.html =404;
```

Comment:
So, is this only telling you which file to load? So, /blabla would still be /blabla, but just resolved by index.html?

Reply:
Correct

Reply:
But existing assets would still be resolved from the file system.